week04/ecommerce/Product_Detail.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Detail</title>
+    <link rel="stylesheet" media="screen and (max-width: 800px)" href="smallstyle.css">
+    <link rel="stylesheet" media="screen and (min-width: 800px)" href="fullstyle.css">
+</head>
+<header>
+    <!-- Navbar -->
+    <nav class="navbar">
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+</header>
+
+<body>
+
+    <h1>Product Details</h1>
+
+    <div> <img src="images/golf_set.jpeg" alt="Golf Set"></div>
+
+    <p>This is where my product detail information will go!</p>
+    <p>Price: $200</p>
+
+    <div class="purchaseForm">
+        <form>
+            <label for="dextarity">Right or Left handed?</label>
+            <input type="radio" name="dextarity" value="RH">RH
+            <input type="radio" name="dextarity" value="LH">LH
+            <label for="purchase">How many would you like to purchase?</label>
```

Comment:
This line is inline currently which is why the word "how" is stuck on the previous line with the radio buttons. Change the display and it will drop down.